docs/content/users/quickstart.md

```diff
@@ -458,19 +458,24 @@ The Laravel project type can be used for [StarterKits](https://laravel.com/docs/
 
 === "OpenMage/Magento 1"
 
-    1. Download OpenMage from [release page](https://github.com/OpenMage/magento-lts/releases).
-    2. Make a directory for it, for example `mkdir ~/workspace/OpenMage` and change to the new directory `cd ~/workspace/OpenMage`.
-    3. Run [`ddev config`](../users/usage/commands.md#config) and accept the defaults.
-    4. Install sample data. (See below.)
-    5. Run [`ddev start`](../users/usage/commands.md#start).
-    6. Follow the URL to the base site.
+    ```bash
+    mkdir my-magento1-site && cd my-magento1-site
+    tag=$(curl -L "https://api.github.com/repos/OpenMage/magento-lts/releases/latest" | docker run -i --rm ddev/ddev-utilities jq -r .tag_name) && curl -L "https://github.com/OpenMage/magento-lts/releases/download/$tag/openmage-$tag.zip" -o openmage.zip
```

Comment:
For M1/OpenMage curl has to be installed.

Reply:
I think we shouldn't hide the output for `curl` in quickstarts, because if people have any problems and look at the output, they won't understand what's going on.

Reply:
> they'll get an error I think

It needs another big `-S` to show errors https://explainshell.com/explain?cmd=curl+-Ss
But let's leave it as it is.

> If we're going to run a docker container we could consider doing all the work there.

It will probably be too much, because we need to:
1. pull `ddev/ddev-utitilies`
2. run it under the same user
3. mount volume

```
docker run -i --rm --pull=always -q --user $(id -u):$(id -g) -v ./:/tmp ddev/ddev-utilities curl ...
```

vs

```
curl ...
```

Reply:
Since OpenMage has explicit support for DDEV, we can probably make this simpler in the future. 

In general, we want to aim for procedures in the quickstarts that are obvious and easy to (visually) parse for any user. OpenMage and Magento2 and Backdrop aren't making that terribly easy for us.